### ISSUE TITLE
[igx-drop-down-item] Aria-label defaults to an ID value

### ISSUE THREAD

**Issue by jamesuhl** [IgniteUI/igniteui-angular#15839]
## Description  
Unless a value for the aria-label is explicitly set, the default behavior of the igx-drop-down component is to set the aria-label to the ID of the `<igx-drop-down-item>` element, which is announced to the screen reader.  

 * igniteui-angular version: 19
 * browser: all

## Steps to reproduce  

1. Add the igx-drop-down component to project. Use default settings.
2. Analyze code in dev tools or use a screen reader. The screen reader will announce the ID when the component receives focus. The image below is from the Infragistics website.
![Image](https://github.com/user-attachments/assets/5a2df72c-e96b-4abe-bdcb-4c14e7064ab3)

## Result  
The screen reader announces the ID

## Expected result  
The screen reader should announce the option

## Solution
The default value should not be an ID. Remove the aria-label from the `<igx-drop-down-item>`. Let the screen reader announce the text inside the `<igx-dropdown-down__inner>`. (Test. This should work). If not, set the aria-label equal to the text inside the `<igx-dropdown-down__inner>` by default.

**Comment by jamesuhl** [comment]
The value of an aria-label should never be an ID. If you're going to default to an ID, then use an aria-labelledby. (but since that would be a self-referencing ID, the 'aria-' attribute is of no use). What needs to happen if [ariaLabel] and [value] are _not_ set  is to not use aria-label at all and let the screen reader announce the text within `<span class="igx-dropdown___inner">`  (I believe that will work--need to test).
